keylime/src/config/env.rs

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Is file header necessary? I have not included it in the latest changes but I doubt if it should be added or not. If so, I will create an additional PR for the ones that don't contain it